### ISSUE TITLE
dilated_block

### ISSUE THREAD

**Issue by TanTingyi** [projectappia/eegnet#1]
WaveNet used a causal conv layer, but there is just normal conv layer in your code.
If it's suitable, or maybe I miss some details?

**Comment by mgsnuno** [comment]
You're correct. I was not using the model as an autoregressor to predict the next input, but instead as a classifier. Also at the time it was not straightforward to apply a causal conv layer. Those were my two reasons.
